spec/post_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative './rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.create(
+      name: 'Loren',
+      photo: 'https://this-person-does-not-exist.com/en/download-page?image=genef4e9868ae582ca3061881b69d8fbeb1',
+      posts_counter: 0,
+      bio: 'I am Loren, and I love sport.'
+    )
+  end
+
```

Comment:
- [ ] It would be better if we implement test for our methods in the models, for example; `update_post_counter` and `recent_comments` should be tested. Kindly do this for the other methods not tested